### ISSUE TITLE
Error on table creation after latest update

### ISSUE THREAD

**Issue by FixTheAdmin** [JorelAli/UltimateCards#2]
Server software: paper-421 1.13.3.
UltimateCards version 1.3.6 (for 1.13.2).
Vault version 1.7.1.
Steps to reproduce:
/table new test 10 poker
/table new test2 10 blackjack
Table is not created. This error is shown in console: https://hastebin.com/oxatakuneq.bash

**Comment by JorelAli** [comment]
I'd recommend in future not using hastebin for bug reports, due to the fact they self-destruct after a short amount of time - I'm glad I anticipated this in advance and took a screenshot of the error:

![image](https://user-images.githubusercontent.com/4613171/50374614-b5134000-05e8-11e9-98ef-874aeab18f26.png)

After looking into the error in the console, the issue is due to the `getId()` method not being present in the `MapView` class supplied by Paper. After looking at the Paper documentation, I can confirm that the [`getId()` does exist](https://papermc.io/javadocs/org/bukkit/map/MapView.html#getId--). When running  build 421 in particular, I was able to confirm that the specific method did exist (although deprecated for some reason) within the downloaded .jar file.

Due to the inconsistencies between the supplied documentation and the actual downloaded .jar file (Documentation doesn't say anything about the method being deprecated), I honestly have no idea what to do in this situation.
